.gitlab-ci.yml

```diff
@@ -0,0 +1,32 @@
+# Mandatory include
```

Comment:
Would be nice to verify that the pipeline works, but since CI is currently stopped I don't see a reason to block this PR. If there are issues once the CI is reactivated they can be resolved later.